app/controllers/application_controller.rb

```diff
@@ -6,4 +6,8 @@ class ApplicationController < ActionController::Base
   def current_user
     @current_user ||= User.find_by(user_id: session[:user_id]) if session[:user_id]
   end
+
+  def require_login
```

Comment:
nice! I'd added a card re: this in our 'come back to later' column, so I'll move that over to 'done' - we should make sure that this is covered by our tests as well though. so I'll keep a separate card for that in our to do's